scripts/EN/Harry

```diff
@@ -1916,9 +1916,9 @@ LoadSecondEffect(0x0, 0x0, 0x0); // 0x21f4: 0000306e
 CleanScreen(); // 0x21fe: 00000064
 LoadBackground(0x16); // 0x2202: 00001069
 DrawBackground(); // 0x2208: 00000065
-ShowText((1092)"My one shot hits the wing, blasting a large hole in it. The creature loses altitude as it falls to the floor."); // 0x220c: 0000100c
+ShowText((1092)"My one shot hits the wing, W1blasting a large hole in it.¥nW1The creature then lost altitude and fell to the floor with a smack."); // 0x220c: 0000100c
```

Comment:
Did a bit more research and found that kuchari-to is more onomatopoeia! I checked out some Japanese sites and they defined it as the sharp sound that happens when something breaks or is crumpled up. So I changed the last sentence to end with "...falls to the floor with a smack." to convey this.